packages/app-project/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "PANOPTES_ENV=${PANOPTES_ENV:-production} next build",
     "build-storybook": "build-storybook -s public",
     "dev": "PANOPTES_ENV=${PANOPTES_ENV:-staging} node server/dev-server.js",
-    "lint": "zoo-standard --fix | snazzy",
+    "lint": "next lint",
```

Comment:
shouldn't we run both linters?

Reply:
Worth trying in a separate PR. I've turned off the linter for `content-pages` because it was stopping the app from building and deploying.